swagger.yaml

```diff
@@ -9129,8 +9173,10 @@ definitions:
         type: string
         format: uuid
 
-  infra-env-image-url:
```

Comment:
does it affect any of the clients?

Reply:
I'm not sure, good question. I hope not as the API isn't actually changing, but maybe it would change the type name that was generated or something.

I don't think the agent is getting pre-signed URLs and given that the tests passed I guess it's not a problem for test-infra. Can we ask someone on the UI team?

Reply:
**It will change the name of the generated TypeScript interface**

TL;DR: no big deal because the object remains the same (same members) it won't affect the UI code during runtime. In our dev-environments, we'll have to rename the interface in a few places.

PS: thanks for marking the `url` as required :)